transport/http/inbound_example_test.go

```diff
@@ -22,6 +22,7 @@ package http_test
 
 import (
 	"fmt"
+	"io"
 	"log"
 	nethttp "net/http"
```

Comment:
I know you didn't do this, but there's no need for the `nethttp` alias and it was confusing to me - just remove it and s/nethttp/http/ everywhere (it's fine that the package here is http itself, I do the same thing in transport/grpc)

Reply:
Sure - guessing it's a conflict w/ the current http package, but that seems wack because we're *in* that package, so it shouldn't conflict.

Reply:
Aaah - because this is documentation, yarpc/transport/http *is* imported.